src/components/table/_table.scss

```diff
@@ -22,6 +22,7 @@
     padding: govuk-spacing(2) govuk-spacing(4) govuk-spacing(2) 0;
     border-bottom: 1px solid $govuk-border-colour;
     text-align: left;
+    vertical-align: top;
```

Comment:
I wonder if we should only vertically align top for cells?

And consider vertically aligning bottom for headers (or doing nothing)?

Reply:
We've just hit this (when converting our existing table templates to the Design System code).

The default of middle is definitely weird - there’s very few circumstances where that looks right.

Our current behaviour is to vertically-align headers to the top, partly because that makes most sense with our 'sortable' indicator, and partly because they sometimes have a parent header above them, which it’s good to be close to.

Eg see:

<img width="990" alt="Screenshot 2019-03-22 at 13 27 26" src="https://user-images.githubusercontent.com/30665/54825844-612afc80-4ca6-11e9-8ca6-fcc9c8732d31.png">

Both top and bottom are valid options though, so whichever one is picked for the default, it would be good to have an override, eg `govuk-table__header--vertical-align-top`. Same for cells probably?